web-console/knikubevirt/Create-vm/step-6-results/wizard-results.md

```diff
@@ -0,0 +1,45 @@
+# Create Virtual Machine Wizard - Results Step
+Once the user has clicked "Create virtual machine" they will see the progress state of the VM being created.
+
+![user clicks create button](img/create-vm-inline-wizard-in-progress.png)
+
+Alternatively, The user could navigate away from this page, In that case they'd see the VM in the list in a starting state. This will rely on the VM successfully being created without issue. For more info on error/success states see below.
+
+![vm is being started](img/vm-starting-state.png)
+
+## Success State
+If the process is successful they will see a success state on the Results page.
+
+There is no navigation provided on this page because the user can not go back and make changes at this point. 
+They are provided actions to either go to the VM details or view the VM list.
+
+![VM success](img/create-vm-inline-wizard-success.png)
+
+If they are looking at the list view they will see a status of `running` or `off` if the creation was successful. 
+
+## Error State
+
+If there is an error during the creation of the VM the results page will show a breakdown of the errors with links to the steps where the error exists. 
+
+They will also see the YAML(s) available for review.
+
+![VM Error](img/create-vm-inline-wizard-error.png)
+
+In the step that contains the error they would see the error state of the input they need to address.
+
+![vm error state](img/create-vm-inline-wizard-input.png)
+
+Alternatively, The user could navigate away from this page, In that case they'd see the VM in the list in a error state. They would then be prompted in the status popover to edit the details and correct the issue. 
```

Comment:
not sure if it would be pretty from UX point of view, but we could also redirect user back to the dialog (the VM might not exist yet); see all the errors and YAMLs and retry creation again.

It would be quite some work, but it is an achievable flow we could do.

Reply:
yes, that's a good point. If they did it via the details then on save the VM would have to be finalized with the adjusted setting. This feels like an edge case b/c from what I've understood it's not likely we'll see users leave this flow. It would be nice to have them sent back into the wizard if they didn't complete the creation process without error.

Reply:
If the VM creation will error out on backend, the VM does not exist and we do not show it in the list. I dont really see a way we could implement this flow without having some really nasty hack on the UI or a backend support for it.

Reply:
yes, I think the best approach is to send them back into the wizard. WDYT?

Reply:
I've updated this to reflect this as an option. This design is a bit forward thinking b/c of the toast notifications but the inline notification could cover this gap until we have toasts.